### PR TITLE
if

### PR DESCRIPTION
**Pull request by jeesupark96**
Screen Shot 2021-12-08 at 12 24 31 PM
Screen Shot 2021-12-08 at 12 27 14 PM
Screen Shot 2021-12-08 at 12 33 04 PM
Screen Shot 2021-12-08 at 12 38 10 PM
Screen Shot 2021-12-08 at 12 41 14 PM
Screen Shot 2021-12-08 at 12 44 59 PM
Screen Shot 2021-12-08 at 12 48 52 PM
Screen Shot 2021-12-08 at 1 13 50 PM

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person.age < 18) && (person.age < 21)) {
+    return false;
+  } else {
+    return true;
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js]: This should always return `false`:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 1 above)
Comment (jeesupark96) [reply javascript-if/main.js]: oh I just realized!!!!

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person.age < 18) && (person.age < 21)) {
+    return false;
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js]: What's this return if `person.age` is at least `21`?

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 2 above)
Comment (jeesupark96) [reply javascript-if/main.js]: I misunderstood the function 
so I added the legal drinking age too 
I will delete now!

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age < 18) {
+    return false;
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js]: What's the return value if `person` is `18` or older?

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**